Assets/Mirror/Runtime/InterestManagement/InterestManagement.cs

```diff
@@ -0,0 +1,133 @@
+﻿// interest management from DOTSNET
+using UnityEngine;
+
+namespace Mirror
+{
+    [DisallowMultipleComponent]
+    [RequireComponent(typeof(NetworkManager))]
+    public abstract class InterestManagement : MonoBehaviour
+    {
+        // don't update every tick. update every so often.
+        public float updateInterval = 1;
+        double lastUpdateTime;
```

Comment:
I assume this is a double to match NetworkTime, can we add a comment to confirm that